### PR TITLE
Add option to only consider top level folders

### LINKED ISSUE #1: Add option to only consider top level folders as commit msg

### PR DESCRIPTION
**Pull request by johnletey**
### Summary

Fixes #1

### Changes

-

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -40,8 +59,16 @@ func update(path string) {
...
+			if top {
+				first := split[0]
+				filesChanged = append(filesChanged, first)
+			} else {
+				last := split[len(split)-1]
+				filesChanged = append(filesChanged, strings.Split(last, ".")[0])
+			}
+		}
+		if top {
+			filesChanged = removeDuplicates(filesChanged)
```
Comment (johnletey) [inline main.go]: @nikitavoloboev Maybe we should run this command regardless if `--top` is passed in?

Comment (nikitavoloboev) [reply main.go]: agree, lets do that

Comment (johnletey) [reply main.go]: done 👍

**Comment by nikitavoloboev** [comment]
Looks nice. Id update readme too :)

**Comment by nikitavoloboev** [comment]
We should also improve the `--help` output but that can be done in another PR.

**Pull request merged**